### DISCUSSION TITLE
Groq Tool Use

### DISCUSSION THREAD

**Discussion by slavakurilyak** [567-labs/instructor#650] [General]
According to [Instructor](https://python.useinstructor.com/hub/groq/#groq-ai): "while Groq AI does not support function calling directly, you can still leverage the MD_JSON mode for structured outputs"

GroqCloud supports [tool use](https://console.groq.com/docs/tool-use)

**Comment by ShravanSunder** [comment]
Looks like there’s a groq llama 3 model and api now.

**Comment by jxnl** [comment]
happy to take a PR!
